examples/transformerconfigs/README.md

```diff
@@ -44,26 +51,30 @@ commonLabels:
 - path: spec/selector/matchLabels
   create: true
   kind: Deployment
-  (etc.)  
+# add additional paths to resource fields
```

Comment:
The comment `# add additional paths to resource fields` is misleading. How about change it to `truncated`.

Reply:
I'll take out the comment. Add back if needed.